src/pages/ModuleEditor/ModuleEditorSchema.ts

```diff
@@ -7,10 +7,11 @@ export const ModuleEditorSchema = object<ModuleForm>({
   name: string().min(3, 'Must at least be 3 characters').required('Required'),
   specialCode : string().required('Required'),
   description: string().required('Required').min(4, 'Required'),
+  disabled: boolean(),
   updatedAt: string().notRequired(),
   createdAt: string().notRequired(),
   id: number(),
-  userId: number(),
+  ownerId: number(),
```

Comment:
again, with no backend changes for this, how does this not break the model?

Reply:
@jhurst502 I looked and the backend is setting the Owner property which sets this. I'm fine with that changing here, thanks for the catch!